.github/workflows/playwright.yml

```diff
@@ -0,0 +1,43 @@
+name: Playwright Tests
+on:
+  push:
+    branches: [main, master]
+  pull_request:
+    branches: [main, master]
+jobs:
+  e2e-tests:
+    timeout-minutes: 60
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        env:
+          PUPPETEER_SKIP_CHROMIUM_DOWNLOAD: "true"
+        with:
+          node-version: "14.x"
+          args: install
+
+      - name: Install dependencies
+        uses: borales/actions-yarn@v2.3.0
+        with:
+          cmd: install # will run `yarn install` command
+
+      - name: Build
+        uses: borales/actions-yarn@v2.3.0
+        with:
+          cmd: build # will run `yarn build` command
+
+      - name: Test
+        uses: mujo-code/puppeteer-headful@v2
+        env:
+          CI: "true"
+        with:
+          args: npx playwright test --reporter=line
```

Comment:
Does `yarn` work here or does it have to be `npx` because some things aren't installed yet?  
Background is that everything was aligned to use `yarn` to be consistent.

Reply:
Yeah, I couldn't make it work using `yarn` to start playwright.
For some reason `yarn` was calling jest instead of playwright, and there was a warning to use `npx` instead.

The only problem right now is that we need to use `headless: false` when we launch the puppeteer browser to test the extension. And GH Actions is not running the e2e tests properly because of that. 
That is why I used: `mujo-code/puppeteer-headful@v2`.

I wonder what error msg GH Actions will show to us. Can we run the tests again?

Reply:
@MiguelMedeiros sorry for the limitation. actually only one approval should be needed per contributor. I've also added `mujo-code/puppeteer-headful@v2` to the allowed actions. 
I had this limited initially when I've setup some repository secrets.